src/components/vertical-divider/vertical-divider.stories.mdx

```diff
@@ -0,0 +1,373 @@
+import StyledSystemProps from '../../../.storybook/utils/styled-system-props';
+
+<Meta
+  title='Design System/VerticalDivider'
+  parameters={{ info: { disable: true }}}
+/>
+
+# VerticalDivider
+Provides a vertical dividing line to make the adjacent components feel distinctly separate from one another.
+
+## Quick Start
+
+To use component, import the `VerticalDivider` and set the padding to align it with the other components.
+
+```jsx
+  import VerticalDivider from 'carbon-react/lib/components/vertical-divider';
+
+  const MyComponent = () => (
+    <VerticalDivider />
+  );
+```
+
+## Visuals
+
+### Default
+By default the VerticalDivider has a height of `100%` and padding of `24px`.
+
+<Preview>
+  <Story name='default'>
+    <div style={{ display: 'inline-flex' }}> 
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} />
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+      <VerticalDivider />
+      <div style={{ height: 56, width: 56, backgroundColor: '#00815D' }} /> 
+    </div>
+  </Story>
+</Preview>
+
+### In a flex container
+It is possible to set a custom height for the component by passing a value to the `h` prop as either a number or a string: 
+if passing a number the height will be set in pixels; if passing a string you will need to ensure you provide the measurement unit.
+
+<Preview>
+  <Story name='in a flex container'>
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: 24 }}>
+      <Icon type='home' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='settings' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='add' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='minus' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='info' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='warning' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='chevron_up' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='chevron_down' />
+      <VerticalDivider h={ 16 } />
+      <Icon type='edit' />
+    </div>
+  </Story>
+</Preview>
+
+### In a non-flex container
+When using the component in a non-flex container you can set the `displayInline` prop to true (default is false). This will 
+add the css attribute `display: inline;` to the component.
+
```

Comment:
For some reason this preview appears with a scrollbar.
![Screenshot 2020-09-23 at 15 45 35](https://user-images.githubusercontent.com/35449967/94028854-267eb480-fdb4-11ea-82b2-0ab7c9bccdf3.png)

Reply:
I added some styling to the parent div to centre it, I guess it's overflowing slightly and that `Story` component applies an overflow css attribute.